src/kubernetes_cleanup_pods/test/test_pods.py

```diff
@@ -27,6 +28,11 @@ def it_calls_pods_obj_delete(mocker, pods_obj):
         mocker.call('pod1'),
         mocker.call('pod2')
       ])
+    def it_refuses_to_delete_more_than_30_pods(mocker, pods_obj):
```

Comment:
Curious if there is a specific reason for this or if it is just a reasonable guard?

Reply:
A guard - to prevent accidentally deleting all pods in the cluster.